### PR TITLE
Add option to remove topbar from the profile page

### PR DESCRIPTION
**Pull request by saade**
### Description

Every auth page has an option to remove the topbar except for the profile page. This PR adds the same configuration option as the other pages.

### CODE REVIEW

File: packages/panels/src/Pages/Auth/EditProfile.php
Comment (danharrin) [inline packages/panels/src/Pages/Auth/EditProfile.php:1]: Please extract a trait to use here and SimplePage, probably HasTopbar

**Review by danharrin — CHANGES_REQUESTED** [review]
As above

**Comment by saade** [comment]
@danharrin Done. Took a step further and extracted `$hasMaxWidth` too. Feel free to revert it.

**Comment by danharrin** [comment]
Thanks!

**Pull request merged**